assets/src/components/parts/janus-navigation-button/NavigationButton.tsx

```diff
@@ -137,7 +137,7 @@ const NavigationButton: React.FC<any> = (props) => {
     if (sSelected === undefined) {
       sSelected = currentStateSnapshot[`stage.${id}.selected`];
     }
-    if (sSelected !== undefined) {
+    if (sSelected !== undefined && typeof sSelected === 'boolean') {
```

Comment:
should we instead use the `parseBoolean` function in case some value like `"false"` or `1` comes in?